trillian_log_api.proto

```diff
@@ -82,9 +82,10 @@ service TrillianLog {
   }
 
   // Returns the latest signed log root for a given tree. Corresponds to the
-  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  The server will
-  // return InvalidArgument if first_tree_size is greater than the
-  // LatestSignedLogRoot available to that server.
+  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  If
+  // first_tree_size > latest SignedLogRoot.tree_size, the server will return
+  // gRPC `codes.Unavailable` with the latest SignedLogRoot attached in the
```

Comment:
BTW, have we checked that these `Details` get correctly propagated through all mapping layers in our internal environment?

Reply:
The plan was to commit this change which only has one use case and test it.

Reply:
Tested and verified.  Fixed several bugs in the process :-)